website/docs/recipes/as-gateway.md

```diff
@@ -14,6 +14,11 @@ You can use GraphQL Mesh as a gateway for your data sources. CLI's `serve` comma
 ```bash
 mesh serve
 ```
+You can also point to a specific directory which contains the documentation
```

Comment:
Instead of "documentation", wouldn't it be better if we call if "source files" or something else? What do you think?

Reply:
You are absolutely correct.